src/app/search/search.component.ts

```diff
@@ -74,10 +75,9 @@ export class SearchComponent implements OnInit {
   }
 
   searchWithParams(searchParams: any) {
-    this.clearResults();
     let routeParams = {tab: this.selectedTab};
     Object.keys(searchParams).forEach(key => {
-      if (searchParams[key] !== undefined) {
+      if (searchParams[key] !== undefined && searchParams[key] !== 'undefined') {
```

Comment:
ngModel matching needs the disabled option value to be "undefined" rather than empty string